CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 - **Feature:** Added `MiniAppView` to create multiple MiniApp views.
 - **Feature:** Added `MiniAppConfig` data class to hold configuration settings for the `MiniAppView`.
 - **Deprecated:** Old `create` and `createWithUrl` interfaces.
+- **Fix:** Made the `MaxStorageSize` Limit in Bytes.
+- **Fix:** Returning `onSuccess` for the Bulk Delete even if no items could be deleted.
+- **Update:** Refactored the variable name to `maxStorageSizeLimitInBytes` and `newMiniAppSdkConfig`
```

Comment:
@gopi-karmakar the last one will be in sample app changelog.

Reply:
Actually, the `maxStorageSizeLimitInBytes` is not the Sample app parameter but it's the `MiniAppSdkConfig` parameter so it's important I think. 
Yes, I agree `newMiniAppSdkConfig`  related to the sample app I can remove this one.

Reply:
yeah right, i have changed my comments, please remove it .

Reply:
@munir-rakuten Updated, Kindly check.